crates/router/src/connector/bambora/transformers.rs

```diff
@@ -45,16 +50,72 @@ pub struct Browser {
 pub struct BamboraPaymentsRequest {
     amount: i64,
     payment_method: PaymentMethod,
+    customer_ip: Option<std::net::IpAddr>,
+    term_url: Option<String>,
     card: BamboraCard,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
```

Comment:
Is Serialize required for this struct?

For response struct's Serialize not required.